ts/screens/services/__tests__/ServiceDetailsScreen.test.tsx

```diff
@@ -35,6 +38,25 @@ describe("ServiceDetailsScreen", () => {
         component.getByText(service.organization_fiscal_code)
       ).toBeDefined();
     });
+
+    it("should render the ogranization's fiscal code", () => {
```

Comment:
It's unclear the difference with [the test above](https://github.com/pagopa/io-app/pull/3487/files#diff-ab11534b5ccaf3963833373014cba7c3edd83bd0bbf7f0e758e58d22218c9ca1R34). 
It would help to see the difference in the test description.

Reply:
I'm updating the descrption the above one checks if the component renders even if the load of service list api call goes on error, this one check if it renders even if the detail load goes on error (if data was previously available)